src/EnergyPlus/DXCoils.cc

```diff
@@ -14655,140 +14655,111 @@ void ReportDXCoil(EnergyPlusData &state, int const DXCoilNum) // number of the c
     // PURPOSE OF THIS SUBROUTINE:
     // Fills some of the report variables for the DX coils
 
-    // Using/Aliasing
-    auto &DXElecCoolingPower = state.dataHVACGlobal->DXElecCoolingPower;
-    auto &DXElecHeatingPower = state.dataHVACGlobal->DXElecHeatingPower;
-    Real64 TimeStepSys = state.dataHVACGlobal->TimeStepSys;
-    using Psychrometrics::RhoH2O;
-
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-    Real64 RhoWater;
-    Real64 Tavg;
-    Real64 SpecHumOut;
-    Real64 SpecHumIn;
```

Comment:
Replaced ` SpecHumOut` and `SpecHumIn` local variables with `DXCoil` object member variables, used  `Psychrometrics::RhoH2O` directly and defined `Tavg` where it is used.